packages/2d/src/components/SVG.ts

```diff
@@ -0,0 +1,684 @@
+import {SignalValue, SimpleSignal} from '@motion-canvas/core/lib/signals';
```

Comment:
An export for this component should also be added to `packages/2d/src/components/index.ts`